hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleAccountState.java

```diff
@@ -178,12 +179,15 @@ public void deserialize(SerializableDataInputStream in, int version) throws IOEx
 		if (version >= RELEASE_0220_VERSION) {
 			numContractKvPairs = in.readInt();
 		}
-		if (version >= RELEASE_0250_VERSION) {
+
+		if (version >= RELEASE_0230_VERSION) {
```

Comment:
Should allowances be also deserialized after 0.25.0 ? Or is this for migrating previewnet?